### PR TITLE
feat: adds platform independent way to close modal screens

### PR DESCRIPTION
**Pull request by dickie81**
## Description
Provides a platform independent method closing a modal screen

Also as a part of this I needed to allow tests to run as if they were on different platforms.

### Checklist for this pull request
- [x] PR is scoped to one task
- [x] Tests included
- One of:
- [x] The PR is not making any UI change
- [ ] The PR is making a UI change but not a product change
- [ ] Screenshots included

### CODE REVIEW

**Review by francois-codes — DISMISSED** [review]
I want to cry... not because of what you did, but because you had to.

this is so ugly...

Only one comment if you want -  we tend to use the `.test` extension instead of `.spec` but that's not a big deal. if the project started like this, lets leave it

File: src/utils/index.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 };
+
+export const closeModalScreen = Platform.select({
+  android: () => {
+    NativeModules.APReactNativeBridge.handleCommand('stop', {});
+  },
+  ios: () => {
+    NativeModules.ZPReactNativeBridgeListener.postEvent(
+      'dismiss_modal_view',
+      { animated: 1 },
```
Comment (Budaa) [inline src/utils/index.js]: Maybe it's good idea to add optional param for this animated flag? Even though it is iOS only?

**Review by Budaa — CHANGES_REQUESTED** [review]
I think that it shouldn't be in the utils, but in a separate folder instead. It looks like utils folder was intended to be used for internal functions because it's not even exposed in the `index.js`.

File: src/utils/index.js (hunk 1 above)
Comment (dickie81) [reply src/utils/index.js]: Think it should be added in further PR. Might be possible in Android too - need to investigate.

**Comment by dickie81** [comment]
Fair point! Not exposed is not going to work. ZappNavigation OK? Need to add the method for opening internal URLs too.

**Comment by Budaa** [comment]
ZappNavigation sounds good to me.

**Review by Budaa — APPROVED** [review]
LGTM

**Review by francois-codes — APPROVED** [review]

**Pull request merged**